extensions/rust/test/colorize-results/test_rs.json

```diff
@@ -210,7 +342,29 @@
 	},
 	{
 		"c": "let",
-		"t": "source.rust keyword.other.rust",
+		"t": "source.rust keyword.control.rust",
+		"r": {
+			"dark_plus": "keyword.control: #C586C0",
+			"light_plus": "keyword.control: #AF00DB",
+			"dark_vs": "keyword.control: #569CD6",
+			"light_vs": "keyword.control: #0000FF",
+			"hc_black": "keyword.control: #C586C0"
+		}
+	},
+	{
+		"c": " ",
+		"t": "source.rust",
+		"r": {
+			"dark_plus": "default: #D4D4D4",
+			"light_plus": "default: #000000",
+			"dark_vs": "default: #D4D4D4",
+			"light_vs": "default: #000000",
+			"hc_black": "default: #FFFFFF"
+		}
+	},
+	{
+		"c": "mut",
```

Comment:
In the old grammar `mut` was `storage.modifier`, is `keyword` more correct?

Reply:
> * storage — things relating to “storage”.
>     * modifier — a storage modifier like static, final, abstract, etc.

Should be `storage.modifier` I think.